src/modules/service-catalog/components/service-catalog-list/ItemThumbnail.tsx

```diff
@@ -0,0 +1,31 @@
+import ShapesIcon from "@zendeskgarden/svg-icons/src/16/shapes-fill.svg";
+import styled from "styled-components";
+import { getColorV8 } from "@zendeskgarden/react-theming";
+import { Avatar } from "@zendeskgarden/react-avatars";
+
+const StyledAvatar = styled(Avatar)<{ size: string }>`
```

Comment:
We can define the `size` prop as `"medium" | "large"` here as well, instead of a simple string